src/chapters/classes/exercises.rst

```diff
@@ -0,0 +1,27 @@
+.. _classes-exercises-part1:
+
+Exercises: Classes and Objects
+==============================
+
+Before you get started, make sure you have forked and cloned the `starter code <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn3and4classes>`_.
+
+#. Open up the file, ``Student.cs``, and add all of the necessary getters and
+   setters. Think about the access level each field and method should have, and
+   try reducing the access level of at least one setter to less than public.
```

Comment:
since the starter code gives this:

public string Name { get; set; } ...

the fields are already set and private - plus they have getters and setters. what about replacing this item to either have the students write the auto-implementing properties themselves? or starting with ``private name ...`` and asking them to update the fields to the auot-implemented properties?

Reply:
I went with the second option!